protocols/bftcosi/bftcosi.go

```diff
@@ -15,13 +14,16 @@ import (
 // VerificationFunction can be passes to each protocol node. It will be called
 // (in a go routine) during the (start/handle) challenge prepare phase of the
 // protocol. The passed message is the same as sent in the challenge phase.
-type VerificationFunction func(Msg []byte) bool
+type VerificationFunction func(Msg []byte, Data []byte) bool
```

Comment:
I don't get what is the difference between `Msg` and `Data`. For me the old function signature was fine, you verify a bunch of bytes... why is that new addition required ?